README.md

```diff
@@ -145,6 +145,17 @@ jobs:
 | `force_pass`                | <span style="color:green"> optional </span>  | Choose whether to force a pass when checks are done.                           |
 | `workers`                   | <span style="color:green"> optional </span>  | The number of checks (one per file) to run in parallel, defaults to 9 |
 
+## Details
+
+### Hidden File Types
+
+If you need to specify a file_types pattern to include hidden files, you'll need a `.*` pattern that is provided in the context of a comma separated list, for example:
+
+```yaml
+file_types: '.*,'
```

Comment:
what happens here if I only want to check the hidden files? (not sure how frequent is this scenario) 
based on this, I will need to exclude the other non-hidden types right? 
To do that, the only option I see is `exclude_files` but that requires full path right? or can I use a pattern?

Reply:
I think if you add this file types filter it would exclude all the other patterns - in the same way doing *.md only includes markdown. But I haven’t tried it recently!

Reply:
Yes, I use this on a dotfiles repo (where it's all "hidden" files) and this is the pattern I use, after guidance in #76.